uicomponent-compose/core/src/main/java/io/github/droidkaigi/feeder/core/util/SessionAlarm.kt

```diff
@@ -0,0 +1,127 @@
+package io.github.droidkaigi.feeder.core.util
+
+import android.app.AlarmManager
+import android.app.Application
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import androidx.core.app.AlarmManagerCompat
+import io.github.droidkaigi.feeder.TimetableItem
+import io.github.droidkaigi.feeder.core.R
+import io.github.droidkaigi.feeder.defaultLang
+import kotlinx.datetime.Instant
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.toJavaLocalDateTime
+import kotlinx.datetime.toLocalDateTime
+import java.time.format.DateTimeFormatter
+import javax.inject.Inject
+
+class SessionAlarm @Inject constructor(private val app: Application) {
```

Comment:
Since we are using `TimetableItem` this year, could you use `TimetableItemAlarm`? 🙇

Reply:
Renamed file 🙇‍♂️
3ff62d5afa71372229b78edb247817c4990dd768